pyorientAssignment.py

```diff
@@ -36,14 +37,28 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
-for record in records:
-	print record.price
 
+min_p= 20000
+max_p= 0
+#sum_p=888888
+sum_p= 0
 
+for record in records:
+    sum_p+=record.price
+    
+    if record.price<min_p:
```

Comment:
after this conditional you should set the current min_p value to the record.price